therapy/etl/chemidplus.py

```diff
@@ -164,21 +164,38 @@ def _load_record(self, batch: BatchWriter, record: Dict):
             instance
         :param therapy.schemas.Drug record: complete drug record to upload
         """
-        concept_id_l = record['concept_id'].lower()
-        for alias in {a.lower() for a in record['aliases']}:
-            batch.put_item(Item={
-                'label_and_type': f'{alias}##alias',
-                'concept_id': concept_id_l,
-                'src_name': SourceName.CHEMIDPLUS.value
-            })
-        if 'label' in record:
+        concept_id_ref = record['concept_id'].lower()
+        aliases = record.get('aliases')
+        if aliases:
+            for alias in {a.casefold() for a in aliases}:
+                batch.put_item(Item={
+                    'label_and_type': f'{alias}##alias',
+                    'concept_id': concept_id_ref,
+                    'src_name': SourceName.CHEMIDPLUS.value,
+                })
+        elif 'aliases' in record:
```

Comment:
Aliases always exists, see [here](https://github.com/cancervariants/therapy-normalization/blob/427a4228080f261b682f5358e59243a1f1c6ece3/therapy/etl/chemidplus.py#L141). 
```suggestion
        else:
```

Reply:
👍